app/models/concerns/findings/reiterations.rb

```diff
@@ -1,14 +1,23 @@
 module Findings::Reiterations
   extend ActiveSupport::Concern
 
+  DEFAULT_TO_S_PRELOADS = [
+    control_objective_item: {
+      review: [:plan_item, :conclusion_final_review]
+    }
+  ]
+
   included do
     scope :repeated,     -> { where     state: Finding::STATUS[:repeated] }
     scope :not_repeated, -> { where.not state: Finding::STATUS[:repeated] }
 
+    scope :with_parent_id, ->(id) { where "? = ANY(#{Finding.table_name}.parent_ids)", id }
```

Comment:
Prefiero el estilo de los muchachos: https://guides.rubyonrails.org/active_record_querying.html#passing-in-arguments

> Using a class method is the preferred way to accept arguments for scopes.

Igualmente, estás teniendo mucha fe en que la migración se va a dar pronto. Esperemos... pero eso lo dejaría "comentado" como TO-DO.

Reply:
Ahí te salio un francocop nuevo jaja

Reply:
Jajaja, esa gema la voy a escribir algún día. Va a tener más líneas que el Kernel.